client/my-sites/plans/current-plan/jetpack-checklist/constants.ts

```diff
@@ -64,8 +68,31 @@ export const JETPACK_SECURITY_CHECKLIST_TASKS: ChecklistTasks = {
 	},
 };
 
-export const JETPACK_PERFORMANCE_CHECKLIST_TASKS: ChecklistTasks = {
-	// No tasks for this checklist yet…
+export const JETPACK_PERFORMANCE_CHECKLIST_TASKS: Readonly< ChecklistTasksetUi > = {
+	jetpack_site_accelerator: {
+		title: translate( 'Site Accelerator' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Site accelerator' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).